packages/patternfly-react/src/components/MessageDialog/MessageDialog.stories.js

```diff
@@ -0,0 +1,198 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+import { withInfo } from '@storybook/addon-info/dist/index';
+import { withKnobs } from '@storybook/addon-knobs';
+import { defaultTemplate } from 'storybook/decorators/storyTemplates';
+import LinkTo from '@storybook/addon-links/react';
+
+import {
+  storybookPackageName,
+  STORYBOOK_CATEGORY
+} from 'storybook/constants/siteConstants';
+import { name } from '../../../package.json';
+
+import MessageDialog from './MessageDialog';
+
+import MessageDialogToggleableOptions, {
+  MessageDialogToggleableOptionsSource
+} from './Stories/MessageDialogToggleableOptions';
+import MessageDialogNondestructiveAction, {
+  MessageDialogNondestructiveActionSource
+} from './Stories/MessageDialogNondestructiveAction';
+import MessageDialogError, {
+  MessageDialogErrorSource
+} from './Stories/MessageDialogError';
+import MessageDialogDeleteConfirmation, {
+  MessageDialogDeleteConfirmationSource
+} from './Stories/MessageDialogDeleteConfirmation';
+import MessageDialogWarning, {
+  MessageDialogWarningSource
+} from './Stories/MessageDialogWarning';
+import MessageDialogQuestion, {
+  MessageDialogQuestionSource
+} from './Stories/MessageDialogQuestion';
+import MessageDialogInfo, {
+  MessageDialogInfoSource
+} from './Stories/MessageDialogInfo';
+import MessageDialogSuccess, {
+  MessageDialogSuccessSource
+} from './Stories/MessageDialogSuccess';
+
+const modalStoryKind = `${storybookPackageName(name)}/${
+  STORYBOOK_CATEGORY.WIDGETS
+}/Modal Overlay`;
+
+const storyDescription = (
+  <React.Fragment>
+    <p>
+      This component is based on the React Bootstrap Modal component. Please
+      visit the following for complete documentation
+    </p>
+    <ul>
+      <li>
+        <a href="https://react-bootstrap.github.io/components/modal/">
```

Comment:
do you mind using `DOCUMENTATION_URL.REACT_BOOTSTRAP_COMPONENT`? just makes these easier to update...